common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility library for working with the edx-enterprise app
+"""
+
+from django.conf import settings
+
+
+def get_enterprise_branding_info(provider_id):
+    """
+    Client API operation adapter/wrapper
+    """
+    if not enterprise_enabled():
+        return None
+    from enterprise import api as enterprise_api
+    return enterprise_api.get_enterprise_branding_info(provider_id=provider_id)
+
+
+def enterprise_enabled():
+    """
+    Returns boolean indication if enterprise app is enabled on not.
+    """
+    return settings.FEATURES.get('ENTERPRISE_APP', True)
```

Comment:
@asadiqbal08 I think it's simple and consistent to go with a feature flag as you suggest, but we should make sure that it doesn't appear in the admin. We should also make sure that the platform works whether it is installed and loaded as a django app or not. I don't have a strong opinion on this though, other than insisting that it doesn't appear in the admin if it's not enabled.